screenshots/Package.swift

```diff
@@ -0,0 +1,7 @@
+// swift-tools-version:5.5
+
+import PackageDescription
```

Comment:
why do the screenshots folder have `Package.swift`?

Reply:
As mentioned in the PR description, it's to slim down and tidy up the distribution package. So, whenever an empty `Package.swift` file gets added to a folder like this, it excludes that folder from the SPM distribution. 

Merchants don't need to download our sample apps or screenshots or scripts to use our SDK. This is the only working way currently.

Reply:
I removed these to avoid future confusion from our merchants' side